### PR TITLE
[docs] Improve docs on how to contribute to the docs

### PR DESCRIPTION
**Pull request by mikesnoeren**
## The Problem/Issue/Bug:
- The documentation on how to contribute to the documentation wasn't really clear.
- The docker command to serve the documentation locally was missing some plugins.

## How this PR Solves The Problem:
- The documentation is now more clear.
- The docker command has been fixed.

<a href="https://gitpod.io/#https://github.com/drud/ddev/pull/4000"></a>

### CODE REVIEW

**Comment by tyler36** [comment]
Perhaps we should remind people nicely to run a spellcheck (something I often forget 😢 )
Or instructions on how to manually trigger one (is this possible?)

**Comment by rfay** [comment]
I see you did it with git this time as opposed to the web UI, congratulations! One little detail: I don't ever recommend using the "master" branch in your fork. Instead, use a branch named for the feature or bug you're working on. You'll never remember what "master" was. I usually name my branches like "yyyymmdd_short_description". That way they sort (I have many, many of them) and I can remember which one was for which. There's nothing wrong with what you did for this PR, just giving you a hint for the future.

**Review by rfay — CHANGES_REQUESTED** [review]
This is looking great! Now I'm going to stretch you by asking you to add another commit. 

When you make a PR, the PR docs are automatically created, for example, on yours. I think you should tell people about this.
Improve_documentation_on_how_to_contribute_to_the_documentation_by_mikesnoeren_·_Pull_Request__4000_·_drud_ddev

**Comment by rfay** [comment]
I'm sure you saw you have a couple of markdownlint problems at https://github.com/drud/ddev/runs/7361884798?check_suite_focus=true

You probably want to explain to people how to check with markdownlint as well before pushing? `make markdownlint`. 

I always do `make staticrequired` to get all of these, but it does mean installing those tools, and the Makefile tells how to install them.

**Comment by rfay** [comment]
Congrats on getting the docs fixed!

File: docs/content/developers/testing-docs.md [hunk 1]
```diff
@@ -1,7 +1,45 @@
...
+This section is designed for people who want to contribute to the DDEV documentation.
+When you are going to make any changes to the documentation it is recommended that you test them locally to see what your changes look like.
 
-It's easiest to [install mkdocs locally](https://www.mkdocs.org/user-guide/installation/) with `pip3 install mkdocs` and `pip3 install -r docs/mkdocs-pip-requirements`, but if mkdocs isn't installed, `make mkdocs-serve` will run a docker command to serve them on `http://localhost:8000`.
+## Fork / clone the DDEV repository
 
-If you don't have `make` on your system, you can easily install it, but you can also just run the docker command that `make mkdocs-serve` runs. `docker run -it -p 8000:8000 -v "$${PWD}:/docs" -e "ADD_MODULES=mkdocs-material mdx_truly_sane_lists mkdocs-git-revision-date-localized-plugin" -e "LIVE_RELOAD_SUPPORT=true"  -e "FAST_MODE=true" -e "DOCS_DIRECTORY=./docs" polinux/mkdocs;`
+To start making changes you need to get a local copy of the DDEV documentation, we recommend you to [fork the DDEV repository](https://github.com/drud/ddev/fork) as that's where the documentation lives.
```
Comment (rfay) [inline docs/content/developers/testing-docs.md:8]:
```suggestion
To start making changes you'll need a local copy of the DDEV documentation, so [fork the DDEV repository](https://github.com/drud/ddev/fork) as that's where the documentation lives.
```

**Review by rfay — CHANGES_REQUESTED** [review]
I just made one minor wording change; all you have to do is click the "commit" button in the PR.

**Comment by rfay** [comment]
Congratulations on your first PR, and number 4000!

**Pull request merged**